<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.10.12 on linux)" generated="2024-07-26T08:10:47.153095" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="/home/victor-machado/.vscode/extensions/robocorp.robotframework-lsp-1.12.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-07-26T08:10:47.168730" elapsed="6.445169"/>
</kw>
<status status="PASS" start="2024-07-26T08:10:47.168413" elapsed="6.445658"/>
</test>
<status status="PASS" start="2024-07-26T08:10:47.153931" elapsed="6.460686"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-07-26T08:10:47.150694" level="WARN">Error in file '/home/victor-machado/.vscode/extensions/robocorp.robotframework-lsp-1.12.0/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
